server/api/crud/pipelines.py

```diff
@@ -433,3 +440,25 @@ def _get_experiment_id_from_run(run: dict) -> str:
             ):
                 return data.get("id", "")
         return ""
+
+    @staticmethod
+    def _filter_runs_by_name(runs: list, project_name: str, target_name: str) -> list:
```

Comment:
I think you don't need the `project_name` parameter - see my other comment on this function.

Reply:
for sure. I removed it for us.